### DISCUSSION TITLE
vuepress-plugin-comment2 如何在vuepress2 使用

### DISCUSSION THREAD

**Discussion by 782042369** [vuepress-theme-hope/vuepress-theme-hope#1590] [Q&A] [answered]
"vuepress": "^2.0.0-beta.36"
 "vuepress-plugin-comment2": "^2.0.0-beta.17"

**Comment by Mister-Hope** [comment, marked as answer]
I believed everything is mentioned in it's docs at https://vuepress-theme-hope.github.io/v2/comment/zh/

**Reply by 782042369** [reply]
按照这个配置不生效
<img width="448" alt="image" src="https://user-images.githubusercontent.com/28727900/158936811-591d17cc-a1fd-405b-8b46-197e9efe437e.png">
![image](https://user-images.githubusercontent.com/28727900/158936828-fe0c3711-9c78-4d02-8484-9e4f802c7a96.png)

**Reply by Mister-Hope** [reply]
You are expected to insert the Comment components in places you like YOURSELF. Different theme have different layouts, there is NO POSSIABLE WAY the plugin could know where should it insert the comment box into with different themes.

![image](https://user-images.githubusercontent.com/33315834/158936972-1ba73741-c502-4f07-a5aa-51b62c095090.png)

**Reply by Mister-Hope** [reply]
`@vuepress/theme-default` is providing related docs, and you should probably have a look at them: https://v2.vuepress.vuejs.org/zh/reference/default-theme/extending.html

Or, you may use `vuepress-theme-hope` for a much simple configuration and other great features.

**Reply by 782042369** [reply]
感谢。
